subprojects/packagefiles/theora/lib/meson.build

```diff
@@ -170,6 +170,8 @@ libtheoradec = library(
   vs_module_defs: '../win32/xmingw32/libtheoradec-all.def',
   c_args: compile_args,
   link_args: theoradec_version_flag,
+  soversion: theoradec_soversion,
+  version: theoradec_libversion,
```

Comment:
I don't think we need both the soversion and the version, do we? The soversion defaults to the first dotted component of the version, which is what most projects do (including this one).

Reply:
Fixed, thanks.